modules/apps/transfer/keeper/grpc_query_test.go

```diff
@@ -245,12 +245,34 @@ func (suite *KeeperTestSuite) TestEscrowAddress() {
 			},
 			true,
 		},
+		{
+			"failure - channel not found due to invalid portID",
```

Comment:
I think I would put it like this, because the code doesn't actually try to find the channel, it fails before that, right?

```suggestion
			"failure - invalid portID",
```

Reply:
I think this is what we cover in the discussion here https://github.com/cosmos/ibc-go/pull/6059#discussion_r1539724255 :)